_posts/2022-11-01-segmentation.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Segmentation Part I"
+author: Jenna Tomkinson
+member: jenna-tomkinson
+tags:
+  - opinions
+  - process
+---
+
+# A Brief Performance Comparison of Popular Cell Segmentation Software
```

Comment:
```suggestion
# A Brief Comparison of Popular Cell Segmentation Software
```

If you clarify "software usability and performance" in the title, then you can drop the word "performance" in the leading H1 heading.

Reply:
👍